python/dgl/distributed/optim/pytorch/sparse_optim.py

```diff
@@ -65,10 +275,20 @@ def step(self):
                 # Note: we cannot skip the gradient exchange and update steps as other
                 # working processes may send gradient update requests corresponding
                 # to certain embedding to this process.
-                idics = th.cat(idics, dim=0) if len(idics) != 0 else \
-                    th.zeros((0,), dtype=th.long, device=th.device('cpu'))
-                grads = th.cat(grads, dim=0) if len(grads) != 0 else \
-                    th.zeros((0, emb.embedding_dim), dtype=th.float32, device=th.device('cpu'))
+                idics = (
```

Comment:
It is better to split changes here (format change) into a new PR (format change PR).

Reply:
How about we keep this change? Because it's automatically formatted by ```black``` and ```autopep```, which is needed and difficult to revert.